src/components/Login.css

```diff
@@ -0,0 +1,79 @@
+body {
```

Comment:
is this main body? If yes, i think, it should not be here, you should locate in App.css